src/components/Mdx/Pre/Code.dark-theme.css

```diff
@@ -0,0 +1,274 @@
+/* stylelint-disable */
```

Comment:
Why did you move the styles out of the sass file and into a css file?  Does `yarn develop` still cactch changes to css files?

Reply:
This is actually an incompatibility issue with https://github.com/wesrice/jest-style-transformers. For some reason, this Scss file fails to compile when transformed by `jest-style-transformers`.  I haven't been able to get to the root cause yet, so I just switched it from scss to css to circumvent the issue, especially since this isn't a scss module and the styles are at a global scope anyways.